spec/models/comment_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Comment do
+  describe 'validations' do
+    it { is_expected.to validate_presence_of :person_id }
```

Comment:
Added some validations before I realized I didn't know what to do about `create`, can remove them if you'd rather have `create` be part of a different story.